code/modules/uplink/uplink_devices.dm

```diff
@@ -74,3 +74,50 @@
 /obj/item/pen/uplink/Initialize(mapload, owner, tc_amount = 20)
 	. = ..()
 	AddComponent(/datum/component/uplink, owner, TRUE, FALSE, null, tc_amount)
+
+// NT Uplink
+/obj/item/ntuplink
+	name = "\improper nanotrasen uplink"
+	icon = 'icons/obj/radio.dmi'
+	icon_state = "radio"
+	item_state = "walkietalkie"
+	desc = "A specialized uplink for authorized Nanotrasen use only."
```

Comment:
```suggestion
	desc = "A specialized uplink for authorized Nanotrasen use only."
```
can be worded better since anyone can acess it

Reply:
only NT is supposed to use it

Reply:
> only NT is supposed to use it legally, but someone can still steal it and use it

I think he means that descriptions are sometimes used an like an OOC comment to let a player know that it would be against the rules to use something in an X way and the description is written in that way.

Like it isn't telling the in character user that this is for NT use only; it is more like telling the player that you better not use it unless you are an ERT because it would be power gaming or something